bindings/ios/MEGARequest.mm

```diff
@@ -252,4 +253,9 @@ - (MEGABannerList *)bannerList {
     return [[MEGABannerList alloc] initWithMegaBannerList:bannerList cMemoryOwn:YES];
 }
 
+- (MEGAHandleList *)megaHandleList {
```

Comment:
Probably we want here an array?

Please check others getters in the MEGARequest (megaStringListDictionary, megaStringDictionary, stringTableArray)

Reply:
To save the handles in an array we must convert them to object type instead of uint64_t (not sure if would be better as NSString or NSNumber). I have seen the same megaHandleList in MEGAChatRequest, so I did same way. I can change it no problem, but please confirm that we want different approach for sdk bindings than chat bindings or if we should update the chant ones.

Reply:
What we want is to facilitate and simplify the code in the application. 
How are you converting the handle lists in the application?

Returning an array of Uint (`-> [UInt64]`) in swift by using a `compactMap`?

If you see that there is no need to change it, don't do it, it was just a suggestion.